handlers/index.go

```diff
@@ -80,3 +66,26 @@ func IndexPageData(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 }
+
+func getSlotVizData(currentEpoch uint64) types.SlotVizPageData {
```

Comment:
This function does not have any error handling. If errors happen, the slot viz will simply not be shown but there is no information, WHAT went wrong.

This is a delicate topic. For example: We might not want to throw an error log if configuration is missing. But if something else goes wrong? What are your thoughts?